### ISSUE TITLE
Call to undefined function get_mesh_sections()

### ISSUE THREAD

**Issue by nate-allen** [linchpinagency/mesh#28]
In the FAQ, under "Adding a function to your page template(s)," it gives this example:

```
<?php
add_action( 'the_content', 'add_multiple_content_sections' );
function add_multiple_content_sections( $the_content ) {
    $the_content .= get_mesh_sections();
    return $the_content;
} ?>
```

This causes the error `Call to undefined function get_mesh_sections()` because the function doesn't exist.

**Comment by desrosj** [comment]
Thanks @nate-allen, this is fixed with 78b6feede4b45e02502ae5eee5ad274d60043605. The function namespacing changed to `mesh_get_sections()`.

**Comment by nate-allen** [comment]
Thanks @desrosj. This brings up another problem though (and I can open a new issue if that's easier), but `mesh_get_sections()` returns an array. It can't concatenate with the `$content` string, the way the FAQ currently shows it.
